training_config/coref_spanbert_large.jsonnet

```diff
@@ -0,0 +1,109 @@
+// Configuration for a coreference resolution model based on:
+//   Lee, Kenton et al. “Higher-order Coreference Resolution with Coarse-to-fine Inference.” NAACL (2018).
+//   + SpanBERT-large
+
+local transformer_model = "SpanBERT/spanbert-large-cased";
+local max_length = 512;
+local feature_size = 20;
+local max_span_width = 30;
+
+local transformer_dim = 1024;  # uniquely determined by transformer_model
+local span_embedding_dim = 3 * transformer_dim + feature_size;
+local span_pair_embedding_dim = 3 * span_embedding_dim + feature_size;
+
+{
+  "dataset_reader": {
+    "type": "coref",
+    "token_indexers": {
+      "tokens": {
+        "type": "pretrained_transformer_mismatched",
+        "model_name": transformer_model,
+        "max_length": max_length
+      },
+    },
+    "max_span_width": max_span_width,
+    "max_sentences": 110
+  },
+  "validation_dataset_reader": {
+    "type": "coref",
+    "token_indexers": {
+      "tokens": {
+        "type": "pretrained_transformer_mismatched",
+        "model_name": transformer_model,
+        "max_length": max_length
+      },
+    },
+    "max_span_width": max_span_width
+  },
+  "train_data_path": std.extVar("COREF_TRAIN_DATA_PATH"),
+  "validation_data_path": std.extVar("COREF_DEV_DATA_PATH"),
+  "test_data_path": std.extVar("COREF_TEST_DATA_PATH"),
+  "model": {
+    "type": "coref",
+    "text_field_embedder": {
+      "token_embedders": {
+        "tokens": {
+            "type": "pretrained_transformer_mismatched",
+            "model_name": transformer_model,
+            "max_length": max_length
+        }
+      }
+    },
+    "context_layer": {
+        "type": "pass_through",
+        "input_dim": transformer_dim
+    },
+    "mention_feedforward": {
+        "input_dim": span_embedding_dim,
+        "num_layers": 2,
+        "hidden_dims": 1500,
+        "activations": "relu",
+        "dropout": 0.3
+    },
+    "antecedent_feedforward": {
+        "input_dim": span_pair_embedding_dim,
+        "num_layers": 2,
+        "hidden_dims": 1500,
+        "activations": "relu",
+        "dropout": 0.3
+    },
+    "initializer": {
+      "regexes": [
+        [".*linear_layers.*weight", {"type": "xavier_normal"}],
```

Comment:
You may want to initialize the gated sum module for the refinement, just as an FYI.